packages/react/src/components/Menu/MenuItem.tsx

```diff
@@ -251,9 +252,18 @@ class MenuItem extends AutoControlledComponent<WithAsProp<MenuItemProps>, MenuIt
         </ElementType>
       </Ref>
     )
-    const maybeSubmenu =
-      menu && active && menuOpen ? (
-        <>
+    const submenuVisible = menu && active && menuOpen
+
+    // TODO: allow the users to define these via props
```

Comment:
These changes won't be part of the final PR, they are just showcasing the usage in a real example